data/js/values/values.js

```diff
@@ -77,4 +83,8 @@
         }
     });
 
+    nextButton.addEventListener('click', function() {
+        // notify addon that we've clicked the next button
+        addon.port.emit('next_selected');
+    });
```

Comment:
Yeah, let's move that `eventListener` block to an external file here https://github.com/mozilla/all-aboard/tree/master/data/js/lib - perhaps something like `common.js`